articles/introduction-to-c#/index .md

```diff
@@ -0,0 +1,212 @@
+
+### Installing C# 
+After you have installed Visual Studio, choose .NET desktop development and click *Modify*.
+![vscode](installation.png)
+
+After the installation, click *Launch* to get started.  
+
+On the new window, choose *Create new project*.
+![newproject](vs1.png)
+
+Once you have selected *Create new project*, choose *"Install more tools and features"* then click *Next*.
+
+![newproject](Newproject2.png)
+
+Choose *Console App (.NET Core)* from the list and click *Next*.
+
+![newproject](Consoleapp.png)
+
+Enter your preferred project name and click *Create*. In this case, I will name my file *HelloWorld*.
+
+![newproject](projectname.png)
+
+Visual Code will automatically generate some code of C# for you.
+
+![newproject](Program.png)
+
+Let's look at the code.
+
+Below is the code that will be generated. It is a *Hello World* program.
+
+```C#
+using System;
+
+    namespace HelloWorld
+        {
+            class Program
+            {
+            static void Main(string[] args){
+                Console.WriteLine("Hello World!");
+            }
+        }
+    }
+```
+
+For you to run the program, press the *F5* key on your keyboard.
+This will compile and run your code. A console window will open with the following output:
+
+```
+Hello World!
+
+C:\Users\Username\source\repos\HelloWorld\HelloWorld\bin\Debug\netcoreapp3.0\HelloWorld.exe (process 13784) exited with code 0.
+To automatically close the console when debugging stops, enable Tools->Options->Debugging->Automatically close the console when debugging stops.
+Press any key to close this window . . .
+```
+
+### C# Syntax
+From the above code that was generated by Visual Studio, let us look at each line of the code:
+
+**Line 1:** 
+
+ ```C#
+ using System;
+ ```
+
+In C#, it means that we can use the classes available in the system. It contains commonly-used types and classes. 
+You can explore more in the [C# docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive).
+
+**Line 2:**
+ ```C#
+ namespace HelloWorld{}
+ ``` 
+
+Namespaces are used in C# to separate and organize code. They are containers consisting of methods, classes, and other namespaces.
+
+**Line 3:**
+ ``` C#
+ class Program{}
+ ```
+
+A class is a container that contains methods and data. Since C# is Object-oriented programming, creating a class is mandatory for each program. In our case, we are creating a class called `Program`.
+
+**Line 4:** 
+```C#
+static void Main(string[] args)
+```
+Execution of a program begins from the `Main` method. Every C# program must have the `Main` method.
+
+**Line 5:**
+ ```C#
+ Console.WriteLine("Hello World!");
+ ```
+
+`Console.WriteLine()` is used to print out text on the console. It prints a string and moves to the next line. In our case, it is used to output *"Hello World"*.
```

Comment:
Instead of saying `it moves to the next line`, You can say `it adds a new line character to the end of the string`. This is more preciese.

Reply:
Also update for Console.Write().